backend/hitas/calculations/max_price.py

```diff
@@ -121,7 +125,7 @@ def calculate_max_price(
             },
             "rooms": apartment.rooms,
             "apartment_type": apartment.apartment_type.value,
```

Comment:
In Apartment endpoint this is only `type` (instead of `apartment_type`)

Reply:
yep, will fix

Reply:
fixed